indy-besu/docs/design/did-methods.md

```diff
@@ -0,0 +1,60 @@
+# DID Methods
+
+Out of box Ledger provides an ability to use one of two supported DID methods: `did:ethr` or `did:indy`.
+
+Contracts implementing both methods are deployed on the network and integrated with `CL Registry`.
+
+Ledger `permission` related modules are implemented in a way to use **account address** but not a DID.
+
+It is up to a User which DID method to use.
+
+> Moreover, users having an appropriate permissions can even deploy contracts adding support for another DID methods 
+> (need to integrate into `CLRegistry`).
+
+## Ethereum DID method: did:ethr
+
+Ethereum DID Method `did:ethr` described in
+the [specification](https://github.com/decentralized-identity/ethr-did-resolver/blob/master/doc/did-method-spec.md).
+
+### DID Syntax
+
+| parameter        | value                                                   |
+|------------------|---------------------------------------------------------|
+| did              | “did:” method-name “:” namespace “:” method-specific-id |
+| method-name      | "ethr"                                                  |
+| namespace        | “testnet”/"mainnet"                                     |
+| network-chain-id | "0x" *HEXDIG                                            |
+| ethereum-address | "0x" 40*HEXDIG                                          |
+| public-key-hex   | "0x" 66*HEXDIG                                          |
+
+Resource identifier refers to either the ethereum address or secp256k1 public key of an identity associated with this
+DID `0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266`.
+
+Example DID: `did:indy2:0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266`
```

Comment:
Do we want to change it to `did:ethr:0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266`
And do we want to describe ethr syntax if we don't implement it?